### PR TITLE
feat: support node-collector commands and NodeInfo 

### PR DESCRIPTION
**Pull request by chen-keinan**
## Description
support node-collector commands

- Related https://github.com/aquasecurity/trivy/issues/6602

Design Doc: https://github.com/aquasecurity/k8s-node-collector/blob/main/docs/spec-design.md

### CODE REVIEW

**Review by nikpivkin — DISMISSED** [review]

**Comment by simar7** [comment]
@chen-keinan it lgtm, but do we have a ID generator for commands like we have for checks? It would help to have such a generator so you can run it and know what's the next available ID that's not yet taken, when writing a new command.

**Comment by chen-keinan** [comment]
> @chen-keinan it lgtm, but do we have a ID generator for commands like we have for checks? It would help to have such a generator so you can run it and know what's the next available ID that's not yet taken, when writing a new command.

@simar7 thanks for the feedback I have added command ID generator

File: cmd/command__id/main.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+func main() {
+	ids, err := GetCommandIDRange()
+	if err != nil {
+		panic(err)
+	}
+	for i := 0; i < len(ids); i++ {
+		if !ids[i] {
+			println(fmt.Sprintf("%s-%04d", "CMD", i+1))
+		}
```
Comment (simar7) [inline cmd/command__id/main.go:25]: I think it's enough to print the next available one. Or else the output is too verbose.

```shell
<snip>
CMD-9985
CMD-9986
CMD-9987
CMD-9988
CMD-9989
CMD-9990
CMD-9991
CMD-9992
CMD-9993
CMD-9994
CMD-9995
CMD-9996
CMD-9997
CMD-9998
CMD-9999
```


See below for how we do it for checks:
```shell
$ make id

The following IDs are free - choose the one for the service you are targeting.

             AWS: AVD-AWS-0344
             AZU: AVD-AZU-0052
          CLDSTK: AVD-CLDSTK-0002
             DIG: AVD-DIG-0010
              DS: AVD-DS-0031
             GCP: AVD-GCP-0069
             GIT: AVD-GIT-0005
             KCV: AVD-KCV-0093
             KSV: AVD-KSV-0123
            KUBE: AVD-KUBE-0003
             NIF: AVD-NIF-0022
             OCI: AVD-OCI-0002
          OPNSTK: AVD-OPNSTK-0006

```

File: cmd/command__id/main.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
+package main
```
Comment (simar7) [inline cmd/command__id/main.go:1]: The directory should be `cmd/command_id/`, I think there's an accidental double underscore.

**Comment by chen-keinan** [comment]
@simar7 comments has been addressed

**Review by simar7 — APPROVED** [review]

**Pull request merged**